### PR TITLE
feat: Messaging user level disable/enable

### PR DESCRIPTION
**Pull request by wadedvsa**
## Description

Caseworkers can disable messaging functionality for a VOL user. This would be used in instances where the user was abusing the system or breaching terms of service.

Related issue: [VOL-4388](https://dvsa.atlassian.net/browse/VOL-4388)

## Before submitting (or marking as "ready for review")

- [ ] Does the pull request title follow the [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) specification?
- [ ] Have you performed a self-review of the code
- [ ] Have you have added tests that prove the fix or feature is effective and working
- [ ] Did you make sure to update any documentation relating to this change?

### CODE REVIEW

File: module/Olcs/config/module.config.php [hunk 1]
```diff
@@ -67,6 +67,11 @@
 use Olcs\Controller\Lva\Factory\Controller\Variation as LvaVariationControllerFactories;
 use Olcs\Controller\Lva\Licence as LvaLicenceControllers;
 use Olcs\Controller\Lva\Variation as LvaVariationControllers;
+use Olcs\Controller\Messages\ApplicationConversationListController;
```
Comment (hobbyhacker0) [inline module/Olcs/config/module.config.php:70]: Not on you, but at somepoint we should refactor these to
`Olcs\Controller\Messaging\*`
to match Commandhandler etc

File: module/Olcs/config/module.config.php [hunk 2]
```diff
@@ -872,6 +878,21 @@
             RouteParam\LicenceFurniture::class,
             RouteParam\Licence::class,
         ],
+        LicenceEnableDisableMessagingController::class => [
+            RouteParam\Conversation::class,
```
Comment (hobbyhacker0) [inline module/Olcs/config/module.config.php:882]: Can we do without Conversation here?

Comment (wadedvsa) [reply module/Olcs/config/module.config.php:882]: What do you mean?

Comment (hobbyhacker0) [reply module/Olcs/config/module.config.php:882]: We'd have to change the path to something organisation based - probably out of scope at this point

**Pull request merged**